TF_Tooling_Web/src/Cursors/tests/SvgCursor.test.ts

```diff
@@ -0,0 +1,118 @@
+import TouchFree from '../../TouchFree';
+import { HandChirality, HandType, InputType, InteractionType, TouchFreeInputAction } from '../../TouchFreeToolingTypes';
+import { SVGCursor } from '../SvgCursor';
+
+TouchFree.Init();
+const svgCursor = new SVGCursor();
+const cursor = document.getElementById('svg-cursor');
+const cursorRing = document.getElementById('svg-cursor-ring');
+const cursorDot = document.getElementById('svg-cursor-dot');
+
+const mockTfInputAction = (input?: Partial<TouchFreeInputAction>) =>
```

Comment:
:shrug: It might be useful in the future to have a general test utilities file as this looks very useful outside of just this test

Reply:
Yes! Was thinking the same, I'll move this to somewhere more generic

Reply:
Done